pinax/likes/auth_backends.py

```diff
@@ -11,7 +11,7 @@ def is_allowed(self, obj):
         return _allowed(obj)
 
     def has_perm(self, user, perm, obj=None):
-        if perm == "phileo.can_like":
+        if perm == "likes.can_like":
```

Comment:
This we can't do, because the auth framework depends on the final app module name and model.